docs/documentation.md

```diff
@@ -0,0 +1,16 @@
+# Documentation 
```

Comment:
Now deployed at https://qibogang.github.io/workflows/introduction, it will be moved to https://qibogang.github.io/workflows/ (currently empty) after renaming